### DISCUSSION TITLE
Jiffy locale initialisation trims country code

### DISCUSSION THREAD

**Discussion by Brazol** [GetStream/stream-chat-flutter#1245] [Q&A] [answered]
When Jiffy locale is initialized inside StreamChatState only languageCode is passed. Is that intended? 

![image](https://user-images.githubusercontent.com/5622717/177544777-c9eff72d-98b2-43eb-94b0-9c01643fbb67.png)

When current locale is "en_gb" the "gb" part is not passed to Jiffy and dates inside chat are displayed in the US format instead of UK. If I change `Jiffy.locale(languageCode);` line to `Jiffy.locale(currentLocale.toString());` it works as expected.

**Comment by imtoori** [comment, marked as answer]
https://github.com/GetStream/stream-chat-flutter/pull/1255
I just opened this pr to fix this issue
Thanks for reporting it
